src/index.html

```diff
@@ -1,11 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>This was fun</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <main>
+      <h1>Leaderboard</h1>
+      <section class="container">
+        <section class="right-column">
+             <strong>Recent scores</strong>
+             <button type="button">Refresh</button>
+          <table>
+            <tr>
+              <td>Name: 100</td>
+            </tr>
+            <tr>
+              <td>Name: 200</td>
+            </tr>
+            <tr>
+              <td>Name: 300</td>
+            </tr>
+            <tr>
+              <td>Name: 400</td>
+            </tr>
+          </table>
```

Comment:
- [ ] good job creating the layout with HTML,  however according to the requirements and as requested by the previous code reviewer,  HTML should be used just to make the layout, to fill in the list you should use ES6 modules, with import and export. See the screenshot below and kindly fix that.

![es6](https://user-images.githubusercontent.com/65852621/146245482-dadcd2f9-7abe-4ffd-9bc7-951aeb8e7df1.png)

Reply:
Hi, thank you for your review. However, this project does not require me to populate the list from JavaScript objects if that's what you mean. If so, I might have missed it. Kindly guide me to the requirement that mentions I should create the list using ES6 modules. Here is the link to the project (https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md).

By virtue of using webpack I am already making use of ES6 import/export. If you take a look at my `index.js` I have made one import of CSS `import './style.css';`. This is ES6 syntax.